### DISCUSSION TITLE
Reading quickly changing Data?

### DISCUSSION THREAD

**Discussion by henfri** [jomjol/AI-on-the-edge-device#2862] [General]
Hello,

great project. Amazing work!
Normally, I reading the meters can certainly be done with a large interval.
However, for my usecase, I need quick reading, as I want to read the Power of my Hot-Water supply ("Freshwater Station").
Here a Video:
https://photos.app.goo.gl/6xkg95YUkTP8mG337

Is that possible?

Best regards,
Hendrik

**Comment by caco3** [comment]
No, that is far to fast. I don't think you get below half a minute or so, depending on the number of digits. Also, since the digits change so fast, the camera might catch only fragments of a number, causing wrong/invalid readings.

You say, you want to read the (electrical) power, why not using a Shelly Plus 1PM or a Shelly 3EM?

**Reply by henfri** [reply]
Thanks for your reply.

It is Not the electrical power, but the Power of the transferred heat.

**Reply by caco3** [reply]
Hmm, then I think its very tricky :(
You could record the CAM stream which would be fast enough and do the post processing on a PC. But that is a completely different project.

**Reply by henfri** [reply]
Yes, I understand. 
But unless there is some existing project for the post-processing on the PC, I am not sure it is worth it..

On the other hand: I suppose that 90% of the energy used is used when the water is running for longer time (shower). And then, also the Power is constant for a longer time.

For those cases, every 30 seconds would be sufficient.

Do you think, that this could work?

Greetings,
Hendrik

**Reply by caco3** [reply]
just give it a try. But according to your video, the whole cycle is less than 30s, so how do you trigger the reading?
